### ISSUE TITLE
Terraform supports for "Additional Info" field for EMR?

### ISSUE THREAD

**Issue by DigitalPig** [terraform-providers/terraform-provider-aws#3158]
Continuing #1178 

In our corporation, we need to add an additional line in the aws cli to start EMR cluster successufully.

```shell
--additional-info '{ "instanceAwsClientConfiguration": { "proxyPort": 8099, "proxyHost": "aws-proxy-prod.company.com" } }'
```
However, there is no way of adding this information into terraform field. I was advised to use `Configuration` field but that is for setup of EMR softwares

Is there a possibility to add this option in the terraform aws provider? Thank you!

**Comment by kirupagaran** [comment]
Is Terraform working on this feature ?. If yes, any tentative release dates ?

I need this --additional-info feature so that we can pass proxy information to EMR cluster.

**Comment by saravanan30erd** [comment]
@DigitalPig Looks like its possible, since aws go sdk have that option: https://docs.aws.amazon.com/sdk-for-go/api/service/emr/#RunJobFlowInput
Will work on it.

**Comment by kirupagaran** [comment]
Thanks @saravanan30erd. Any guess on how long it will take to get this feature?

**Comment by kirupagaran** [comment]
Thanks @saravanan30erd. What should I do If I want to use your changes before it gets officially released ? Any steps to build my own terraform linux executable with your changes ?

**Comment by saravanan30erd** [comment]
@kirupagaran Sure, pls contact me here https://www.linkedin.com/in/saravanan30erd/. I will explain the steps.

**Comment by bflad** [comment]
The `additional_info` argument will release with v1.20.0 of the AWS provider, likely later today.

**Comment by bflad** [comment]
This has been released in [version 1.20.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1200-may-23-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
